apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-assigned-services/resource-assigned-services.component.html

```diff
@@ -0,0 +1,24 @@
+<h1 class="page-subtitle">{{'RESOURCE_DETAIL.ASSIGNED_SERVICES.TITLE' | translate}}</h1>
+<perun-web-apps-refresh-button (refresh)="loadAllServices()"></perun-web-apps-refresh-button>
+<button mat-flat-button color="accent" (click)="addService()">
+  {{'RESOURCE_DETAIL.ASSIGNED_SERVICES.ADD_SERVICE' | translate}}
+</button>
+<button mat-flat-button color="warn" class="ml-2" [disabled]="selected.selected.length === 0" (click)="removeServices()">
+  {{'RESOURCE_DETAIL.ASSIGNED_SERVICES.REMOVE_SERVICE' | translate}}
+</button>
+<app-debounce-filter
+  [placeholder]="'RESOURCE_DETAIL.ASSIGNED_SERVICES.FILTER_SERVICES'"
+  (filter)="applyFilter($event)">
+</app-debounce-filter>
+<mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+<div class="mt-3" *ngIf="!loading && assignedServices.length !== 0">
+  <app-services-list
+    (page)="pageChanged($event)"
+    [pageSize]="pageSize"
+    [services]="filteredServices"
+    [selection]="selected">
+  </app-services-list>
+</div>
+<app-alert *ngIf="assignedServices.length === 0 && !loading" color="warn">
```

Comment:
this alert is already part of the service-list, so there is no need to have it here